_includes/header-navigation.html

```diff
@@ -24,6 +24,9 @@
               <li>
                 <a href="https://blog.kie.org/" target="_blank">Blog</a>
               </li>
+              <li>
+                <a href="{{site.baseurl}}/trustyai/" class="{% if page.url contains '/trustyai/' %}active{% endif %}">TrustyAI</a>
```

Comment:
```suggestion
                <a href="{{site.baseurl}}/trustyai/" class="{% if page.url contains '/trustyai/' %}active{% endif %}">Trusty AI</a>
```

Reply:
I don't think we have a "formal" name but I always refer to the initiative as "TrustyAI" so I prefer to keep it as single word